tests/test

```diff
@@ -24,7 +24,7 @@ if [ $(basename $(sudo readlink /proc/1/exe)) == systemd ] ; then
 fi
 
 STAGE1_SRC_FROM_HOST=0
-if tar tvf ../bin/stage1.aci rootfs/flavor|grep -q usr-from-host ; then
+if tar xOvf ../bin/stage1.aci manifest|grep -q usr-from-host ; then
```

Comment:
PEBKAC, sorry.

On Wed, Jun 10, 2015 at 2:24 PM, Dimitri John Ledkov <
notifications@github.com> wrote:

> In tests/test
> <https://github.com/coreos/rkt/pull/1011#discussion_r32168265>:
>
> > @@ -24,7 +24,7 @@ if [ $(basename $(sudo readlink /proc/1/exe)) == systemd ] ; then
> >  fi
> >
> >  STAGE1_SRC_FROM_HOST=0
> > -if tar tvf ../bin/stage1.aci rootfs/flavor|grep -q usr-from-host ; then
> > +if tar xOvf ../bin/stage1.aci manifest|grep -q usr-from-host ; then
>
> not sure about the fonts, it should be 'capital oh' rather than 'digit
> zero'. hopefully the markdown formatted example below should be
> copy&pastable into terminal and working correctly:
>
> cd `mktemp -d`
> echo foo > bar
> tar czf foo.tgz bar
> rm bar
> tar xvOf foo.tgz bar
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/rkt/pull/1011/files#r32168265>.
>